src/components/ContactCard/ContactFormField.jsx

```diff
@@ -1,25 +1,37 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Field } from "react-final-form";
+import { FieldArray } from "react-final-form-arrays";
```

Comment:
I am not aware of `react-final-form-arrays`, it would be cool if you could present it at a crosspoll :)